lib/zinnia.js

```diff
@@ -171,14 +184,16 @@ export async function run ({
             type: 'info',
             message: 'Updated Zinnia module source files, restarting...'
           })
-          childProcess.kill()
+          for (const childProcess of childProcesses) {
+            childProcess.kill()
+          }
```

Comment:
For simplicity, always tear down all processes, instead of only recreating the one that needs to quit